x11-misc/rofi-calc/rofi-calc-1.5.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit eutils autotools
```

Comment:
`eutils` doesnt seem to be used in this ebuild?

Same comments apply to -9999.